examples/speaker_tasks/diarization/README.md

```diff
@@ -186,3 +208,43 @@ Example: `./demo_asr_output/pred_rttms/my_audio1.txt`
 [00:13.97 - 00:15.78] speaker_1: but it's the fourth of july mm
 [00:16.90 - 00:21.80] speaker_0: so yeahg people still work tomorrow do you have to work tomorrow did you drive off yesterday
 ```
+ 
+### Optional Features for Speech Recognition with Speaker Diarization
+ 
+#### Beam Search Decoder
```

Comment:
Is it possible to move beam search decoder related stuff to ASR for wider usage?

Reply:
Once ASR team incorporates pyctcdecode, ctcdecode (Baidu's implementation) will be replaced with pyctcdecode so it will be automatically done.